frontend/src/libs/components/dropdown-menu/dropdown-menu.tsx

```diff
@@ -35,12 +35,17 @@ const DropdownMenu: React.FC<Properties> = ({ routes }) => {
         )}
       >
         {routes.map((item) => {
+          const isSelected = isRouteSelected({
```

Comment:
Prefer quality criteria here

Reply:
@v0ldemar01  do you mean better to name it getSelectedRoute?

Reply:
I think naming is good at now, maybe `getIsSelectedRoute` will be better option

Reply:
getIsSelectedRoute => checkIsSelectedRoute

Reply:
ohhh ok, will change it